src/platform/ESP32/ConnectivityManagerImpl_WiFi.cpp

```diff
@@ -1107,6 +1108,23 @@ void ConnectivityManagerImpl::OnStationIPv6AddressAvailable(const ip_event_got_i
     event.Type                           = DeviceEventType::kInterfaceIpAddressChanged;
     event.InterfaceIpAddressChanged.Type = InterfaceIpChangeType::kIpV6_Assigned;
     PlatformMgr().PostEventOrDie(&event);
+
+#if CONFIG_ENABLE_ENDPOINT_QUEUE_FILTER
+    uint8_t station_mac[6];
+    esp_wifi_get_mac(WIFI_IF_STA, station_mac);
```

Comment:
this returns a esp_err_t ... should it be checked?

Reply:
Added the checked